expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -592,7 +592,7 @@ private void initializeServices() {
         resourceService = new ResourceService(client, CLOCK);
         messageService = new MessageService(client);
         ticketService = new TicketService(client);
-        uriRetriever = new UriRetriever();
```

Comment:
It can't be right that we should use a default http-client and do calls to the internet?

Reply:
It isn't right :) so a good catch. Typically, we use URI in with example.org, which returns HTML and forces you to think…this annoys some people, but it is good practice since the fact you get HTML breaks your test even when you've messed up like this.